src/main/java/org/cactoos/iterator/Partitioned.java

```diff
@@ -72,7 +72,7 @@ public boolean hasNext() {
         if (this.size < 1) {
             throw new IllegalArgumentException("Partition size < 1");
         }
-        return Collections.unmodifiableList(
+        return new Immutable<>(
             new LinkedList<>(
```

Comment:
@andreoss can you remove the wrapping in `LinkedList`, it is now useless, `ListOf` already creates a new list

Reply:
@victornoel Done